projects/media-viewer/src/lib/media-viewer/viewers/pdf-viewer/pdf-viewer.component.ts

```diff
@@ -84,6 +88,21 @@ export class PdfViewerComponent implements AfterViewInit {
       this.pdfWrapper.downloadFile(this.url, this.downloadFileName);
     }
   }
+
+  @Input()
+  set changePage(operation: ChangePageOperation | null) {
+    if (operation) {
+      if (operation.changePageParameter instanceof SetCurrentPage) {
+        this.pdfViewer.currentPageNumber = operation.changePageParameter.pageNumber;
+      } else if (operation.changePageParameter instanceof ChangeByDelta) {
```

Comment:
we're kind of ending up with switch statement like behaviour again - perhaps it should be two different operations?